modules/nixos/installation/base.nix

```diff
@@ -1,15 +1,36 @@
 { pkgs, config, lib, inputs, system, ... }:
 {
-  boot.kernelPackages = config.boot.zfs.package.latestCompatibleLinuxPackages;
-  environment.sessionVariables.NIXPKGS_ALLOW_UNFREE = "1";
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
+  #
+  # WARNING: this is dangerous for systems
+  # outside the installation-cd and shouldn't
+  # be used anywhere else.
+  security.polkit.extraConfig = ''
```

Comment:
Maybe it is worth only setting this if the `format` (module arg that snowfall lib passes) is iso?